app/src/main/java/com/android/bookswap/data/source/api/ApiService.kt

```diff
@@ -0,0 +1,11 @@
+package com.android.bookswap.data.source.api
+/*
+  This interface is used to send a chat request to the API
+*/
+interface ApiService {
```

Comment:
While it is a minute problem, I had a few issues understanding what this function was for, maybe give an example in a comment above